src/utils.py

```diff
@@ -0,0 +1,15 @@
+import os
+
+
+def setup_env():
+    # replace RPi.GPIO with fake GPIO if environment is DEV
+    if os.getenv("ENV") != "PROD":
+        print("Running in dev mode")
```

Comment:
Can you merge Logger change to your branch and change this print to a log statement instead ?

Reply:
Merged